csharp/Platform.Ranges/Range[T].cs

```diff
@@ -19,36 +19,48 @@ public struct Range<T> : IEquatable<Range<T>>
         private static readonly EqualityComparer<T> _equalityComparer = EqualityComparer<T>.Default;
 
         /// <summary>
-        /// <para>Returns minimum value of the range.</para>
-        /// <para>Возвращает минимальное значение диапазона.</para>
+        /// <para>Represents a minimum value of the range.</para>
+        /// <para>Представляет минимальное значение диапазона.</para>
         /// </summary>
         public readonly T Minimum;
 
         /// <summary>
-        /// <para>Returns maximum value of the range.</para>
-        /// <para>Возвращает максимальное значение диапазона.</para>
+        /// <para>Represents a maximum value of the range.</para>
+        /// <para>Представляет максимальное значение диапазона.</para>
         /// </summary>
         public readonly T Maximum;
 
         /// <summary>
-        /// <para>Initializes a new instance of the Range class.</para>
-        /// <para>Инициализирует новый экземпляр класса Range.</para>
+        /// <para>Initializes a new instance of the <see cref="Range{T}"/> structure.</para>
+        /// <para>Инициализирует новый экземпляр структуры <see cref="Range{T}"/>.</para>
         /// </summary>
-        /// <param name="minimumAndMaximum"><para>Single value for both Minimum and Maximum fields.</para><para>Одно значение для полей Minimum и Maximum.</para></param>
+        /// <param name="minimumAndMaximum">
+        /// <para>A single value for the minimum and maximum value of the range.</para>
```

Comment:
```suggestion
        /// <para>A single value for the minimum and maximum values of the range.</para>
```
or
```suggestion
        /// <para>A single value for both minimum and maximum values of the range.</para>
```
or
```suggestion
        /// <para>A single value for both minimum and maximum values.</para>
```
or the one I like the most:
```suggestion
        /// <para>A single value for both <see cref="Minimum"/> and <see cref="Maximum"/>.</para>
```

Reply:
Why are you missing `the` article before `both`?